spec/requests/categories_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'Category', type: :request do
+  context 'GET root' do
+    it 'returns category list' do
+      get root_path
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  context 'GET category_bills_url' do
+    it 'returns a category details with bills' do
+      category = FactoryBot.create(:category)
+      get category_path(category)
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  context 'GET category_new_url' do
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly `./spec/requests/bills_spec.rb` and `./spec/requests/categories_spec.rb` files didn't pass, please fix failed `specs` 🙏🏼 

#### Failed examples:
```ruby
rspec ./spec/requests/bills_spec.rb:5 # Bills GET new_category_bill_path returns a 200 response
rspec ./spec/requests/categories_spec.rb:5 # Category GET root returns category list
rspec ./spec/requests/categories_spec.rb:12 # Category GET category_bills_url returns a category details with bills
rspec ./spec/requests/categories_spec.rb:20 # Category GET category_new_url returns a new category form
```

![image](https://github.com/anbehindY/Budget-tracker/assets/63915024/0eaf5a97-913e-4050-85be-08f387ab4451)

Reply:
Tests are running properly from my side though

![Screenshot (83)](https://github.com/anbehindY/Budget-tracker/assets/110251622/f07a6fe1-4807-4b13-86f0-a464b525f0e0)